README.md

```diff
@@ -13,7 +13,7 @@ You can [view a collection of auto-generated examples](http://alphagov.github.io
 
 ## Requirements
 
-The Ruby language (1.9.3+), the build tool [Rake](http://rake.rubyforge.org/) & the dependancy management tool [Bundler](http://bundler.io/)
```

Comment:
Hi @robinwhittleton, I noticed that 'webjar_packager.rb' attempts to create a jar file. That's why I added Java as a requirement. I had to install Java to make this work.